prisma/schema/TeamMatch.prisma

```diff
@@ -0,0 +1,49 @@
+model TeamMatch {
+  id_num                     Int              @id @default(autoincrement())
+  match_key                  Int
+  team                       Team             @relation(fields: [team_key], references: [key])
+  team_key                   Int
+  TeleActions                TeleActionData[]
+  AutoActions                AutoActionData[]
+  center_auto                Boolean?
+  auto_mobility              Boolean?
+  auto_score_grass           Int?
+  auto_score_feeding_station Int?
+  tele_score_grass           Int?
+  tele_score_feeding_station Int?
+  tele_score_robot           Int?
+  skill                      Int?
+  notes                      String?
+  incap_time                 Int?
+  scoutId                    Int?
+  scout                      User?            @relation(fields: [scoutId], references: [id])
```

Comment:
I don't think we need both of these field? Unless this is something prisma generated?